SUBMISSION-GUIDELINES.md

```diff
@@ -2,43 +2,71 @@
 
 There are two ways to submit your activities. The primary method is to use the [website](#website-form), whereas the secondary is to send by [email](#email) to `founding.members (at) jsgenesis.com`.
 
-What they both have in common is that you provide a .txt file that should be structured like this:
+Note that in either case, make sure you include all your activities up until the end of the scoring round. When you have submitted for round `n`, you can no longer include any activity that **took place before the end of that round**. Hence, anything that was concluded in round `n-1` or `n`, will not be accepted. This rule may sound strict, but if not, it would be very difficult for us to spot if something was submitted twice...
```

Comment:
```suggestion
Note that in either case, make sure you include all your activities up until the end of the scoring round. When you have submitted for round `n`, you can no longer include any activity that **took place before the end of that round**. Hence, anything that was concluded in round `n-1` or `n`, will not be accepted. This rule is to prevent us awarding points for the same activity more than once.
```